stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -54,6 +59,10 @@
   @ParametersDelegate
   private FuseIOParameters mParameters = new FuseIOParameters();
 
+  /** Names of the directories created by workers for the test, also unique ids of the workers. */
+  private List<String> mWorkerDirNames;
+  /** 0-based id of this worker/job worker. */
+  private int mWorkerZeroBasedId;
```

Comment:
For Fuse stressbench, all it knows is the job worker's hostname and id.
And it knows which local folder or Fuse mount point it's write/list/read to/from.

So this job worker, this stressbench test doesn't know whether this node has a worker with Fuse, or a standalone Fuse, or nothing at all

Try to use more precise name